### PR TITLE
[Resource]Add host_id to SPP

### PR DESCRIPTION
**Pull request by michaelkad**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TESTARGS='-run=TestAccXXX'

...
```

### CODE REVIEW

File: website/docs/r/pi_shared_processor_pool.html.markdown [hunk 1]
```diff
@@ -22,7 +24,8 @@ resource "ibm_pi_shared_processor_pool" "testacc_shared_processor_pool" {
 }
 ```
 
-**Note**
+## Notes
```
Comment (ismirlia) [inline website/docs/r/pi_shared_processor_pool.html.markdown:27]: Please double check this when you get a chance, but I think ### was equivalent to **. I recall there being some size differences when I tried it yesterday.

```suggestion
### Notes
```

File: website/docs/r/pi_shared_processor_pool.html.markdown [hunk 2]
```diff
@@ -41,45 +44,49 @@ resource "ibm_pi_shared_processor_pool" "testacc_shared_processor_pool" {
...
+* `shared_processor_pool_id` - (String) The shared processor pool's unique ID.
+* `status` - (String) The status of the shared processor pool.
+* `status_detail` - (String) The status details of the shared processor pool.
 
 ## Import
 
 The `ibm_pi_shared_processor_pool` resource can be imported by using `power_instance_id` and `shared_processor_pool_id`.
 
-**Example**
+## Example
```
Comment (ismirlia) [inline website/docs/r/pi_shared_processor_pool.html.markdown:88]: Same thing here
```suggestion
### Example
```

File: website/docs/r/pi_shared_processor_pool.html.markdown [hunk 3]
```diff
@@ -41,45 +44,49 @@ resource "ibm_pi_shared_processor_pool" "testacc_shared_processor_pool" {
...
+  * `status` - (String) Status of the instance.
+  * `uncapped` - (Bool) Identifies if uncapped or not.
+  * `vcpus` - (Float) The amout of vcpus for the server instance.
+* `shared_processor_pool_id` - (String) The shared processor pool's unique ID.
+* `status` - (String) The status of the shared processor pool.
+* `status_detail` - (String) The status details of the shared processor pool.
 
 ## Import
 
 The `ibm_pi_shared_processor_pool` resource can be imported by using `power_instance_id` and `shared_processor_pool_id`.
```
Comment (ismirlia) [inline website/docs/r/pi_shared_processor_pool.html.markdown:86]:
```suggestion
The `ibm_pi_shared_processor_pool` resource can be imported by using `pi_cloud_instance_id` and `shared_processor_pool_id`.
```

File: website/docs/r/pi_shared_processor_pool.html.markdown [hunk 4]
```diff
@@ -41,45 +44,49 @@ resource "ibm_pi_shared_processor_pool" "testacc_shared_processor_pool" {
...
-- `shared_processor_pool_id` - (String) The shared processor pool's unique ID.
-- `status` - (String) The status of the shared processor pool.
-- `status_detail` - (String) The status details of the shared processor pool.
+  * `availability_zone` - (String) Availability zone for the server instances.
+  * `cpus` - (Integer) The amount of cpus for the server instance.
+  * `id` - (String) The server instance ID.
+  * `memory` - (Integer) The amount of memory for the server instance.
+  * `name` - (String) The server instance name.
+  * `status` - (String) Status of the instance.
+  * `uncapped` - (Bool) Identifies if uncapped or not.
```
Comment (ismirlia) [inline website/docs/r/pi_shared_processor_pool.html.markdown:78]:
```suggestion
  * `uncapped` - (Boolean) Identifies if uncapped or not.
```

Are we changing all references to Bool to Boolean?

Comment (michaelkad) [reply website/docs/r/pi_shared_processor_pool.html.markdown:78]: good catch!

**Review by ismirlia — APPROVED** [review]

**Comment by michaelkad** [comment]
Merged

**Pull request closed without merging**